store/memberSlice.ts

```diff
@@ -11,8 +11,15 @@ export interface Member {
   name: string;
   role: string;
   avatarUrl: string;
-  websiteUrl: string;
+  year: string;
   executive: string;
+  markdown?: string;
+  links: {
+    Website: string | null;
+    Linkedin: string | null;
+    Instagram: string | null;
+    Github: string | null;
```

Comment:
same thing no in our use case?

```suggestion
    Website?: string;
    Linkedin?: string;
    Instagram?: string;
    Github?: string;
```

Reply:
If there's no value it becomes null, I think the ? adds the undefined case but not the null case

Reply:
yeah thats correct, dont think we need to be this explicit about it could just do `if (links.website) {...}` kinda thing

![image](https://github.com/utmmcss/mcss-website-frontend/assets/70813519/58dc4228-68e8-4d5c-83b7-e8ab3f3195cd)

Reply:
theres also this operator if you didnt know https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing which we could use to assign to the redirect in this case but no need to change it,

still should change the types tho